legislators-social-media.yaml

```diff
@@ -4936,3 +4936,9 @@
     youtube_id: UCVroMJWH3jsh4shPSlp24sw
     twitter_id: 346509049
     instagram: michael.fq.san.nicolas
+- id:
+    bioguide: L000594
+    govtrack: 456790
```

Comment:
I think you need the `social:` section here (and then indent the following lines).